diff --git a/node_modules/expo-dev-launcher/android/build.gradle b/node_modules/expo-dev-launcher/android/build.gradle
index 3cc1b3c..c10234f 100644
--- a/node_modules/expo-dev-launcher/android/build.gradle
+++ b/node_modules/expo-dev-launcher/android/build.gradle
@@ -11,7 +11,7 @@ buildscript {
 
 apply plugin: 'com.android.library'
 apply plugin: 'expo-module-gradle-plugin'
-apply plugin: 'org.jetbrains.kotlin.plugin.compose'
+// apply plugin: 'org.jetbrains.kotlin.plugin.compose'
 apply plugin: 'org.jetbrains.kotlin.plugin.serialization'
 apply plugin: 'com.apollographql.apollo'
 
